sources/platform/console/two-factor-authentication.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Two-factor authentication
+description: Learn about Apify Console's two-factor authentication process and how to set it up.
+sidebar_position: 5
+category: platform
+slug: /console/two-factor-authentication
+---
+
+**Learn about Apify Console's account two-factor authentication process and how to set it up.**
+
+---
+
+If you use your email and password to sign in to Apify Console, you can enable two-factor authentication for your account. This will add an extra layer of security to your account and will prevent anyone who gains access to your password from signing in to your account.
```

Comment:
```suggestion
If you use your email and password to sign in to Apify Console, you can enable two-factor authentication for your account. This will add an extra layer of security to your account and prevent anyone who gains access to your password from signing in to your account.
```